src/EventBuilder/Templates/StaticTemplate.mustache

```diff
@@ -0,0 +1,19 @@
+﻿
+{{#Namespaces}}
+namespace {{Name}}
+{
+    public static class Events
```

Comment:
So I noticed while using the ReactiveUI.Events.XamEssentials package that the static class that allows access to the Events is `Events`.  I am thinking it should be a bit more specific to the domain it is surfacing events for.  I know this would be a breaking change for this library, but it opens us up for further Events in the future.